src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/CreateWrapperOfTypeTests.Windows.cs

```diff
@@ -0,0 +1,117 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices.Tests.Common;
+using Xunit;
+
+namespace System.Runtime.InteropServices.Tests
+{
+    public partial class CreateWrapperOfTypeTests
```

Comment:
can change the nameof the file to nonNetfxwindows and compile only on nonnetfx platforms.

Reply:
Do we have other examples of tests like that? I have only observed `*windows.cs` files not any more specific. I can appreciate the targeted focus of some of this, because the more we do this the more the logic is placed into file names instead of attributes - which is literally the point of the attribute. What is the preference for this change?

Reply:
https://github.com/dotnet/corefx/blob/master/src/System.Runtime.Extensions/tests/System/IO/PathTests_Windows_NetFX.cs

Reply:
I see. Unfortunately event that example is always included and has the attribute in it as well. I would prefer to leave this as is since there is clearly no consistency about how this is done - even in that project. The `*.windows.cs` is definitely a common concept and is even documented in `coreclr`.

Reply:
Keeping this as-is